utils/build-script-impl

```diff
@@ -1836,7 +1842,7 @@ function set_swiftsyntax_build_command() {
         --build-dir="${build_dir}"
         --swift-build-exec="${SWIFT_BUILD}"
         --swift-test-exec="${SWIFT_TEST}"
```

Comment:
Don't `--swift-build-exec=` and `--swift-test-exec=` need to change as well ?

Reply:
How about not passing these options at all and on swift-syntax's build script just do a `swift build` or `swift test` invocation if these options are missing ?